pages/linux/du.md

```diff
@@ -1,23 +1,27 @@
 # du
 
-> Estimate file space usage.
+> Disk usage: estimate and summarize file and folder space usage.
 
-- Get a sum of the total size of a file/folder in human readable units:
+- List the sizes of a folder and any subfolders, in the given unit (B/KB/MB):
 
-`du -sh {{file_or_directory}}`
+`du -{{b|k|m}} {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in KB:
+- List the sizes of a folder and any subfolders, in human-readable form (i.e. auto-selecting the appropriate unit for each size):
 
-`du -k {{file_or_directory}}`
+`du -h {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in MB:
+- Show the size of a single folder, in human readable units:
 
-`du -m {{file_or_directory}}`
+`du -sh {{path/to/folder}}`
 
-- Get recursively, individual file/folder sizes in human readable form:
+- List the human-readable sizes of a folder and of all the files and folders within it (recursively):
```

Comment:
Purely to make the sentence length and structure less daunting. It is semantically an annotation, rather than something that needs to be part of the sentence, so marking it as such gives a visual clue that it doesn't need to be loaded into working memory in order to process the preceding sentence. I don't know, I figure it makes it easier to read. Does that make any sense or am I blabbering? :P